docs/usage/installation.rst

```diff
@@ -37,6 +37,7 @@ Using PyPI (stable version)
 
    The ``plothist`` package can now be imported in any of your scripts. Now you can go on with the :ref:`basics-1d_hist-label`.
 
+To update ``plothist`` to its latest stable version, follow the instructions above, adding the argument ``--upgrade`` right after ``pip3 install``.
```

Comment:
Maybe add a `.. code-block:: python` so the user can just copy/paste?

Reply:
done